src/Simplify.Web.Tests/Model/Validation/Attributes/RangeAttributeTests.cs

```diff
@@ -0,0 +1,82 @@
+using System;
+using NUnit.Framework;
+using RangeAttribute = Simplify.Web.Model.Validation.Attributes.RangeAttribute;
+
+namespace Simplify.Web.Tests.Model.Validation.Attributes;
+
+[TestFixture]
+public class RangeAttributeTests : AttributesTestBase
+{
+	public const int MinValue = 2;
+	public const int MaxValue = 12;
+
+	[OneTimeSetUp]
+	public void SetupAttribute() => Attr = new RangeAttribute(MinValue, MaxValue);
+
+	[Test]
```

Comment:
Please add a test which check comparison between different types, for example int and long

Reply:
I have already added it on line 68